bandwidth/model/bxml/verbs/speak_sentence.py

```diff
@@ -0,0 +1,42 @@
+"""
+speak_sentence.py
+
+Bandwidth's SpeakSentence BXML verb
+
+@copyright Bandwidth INC
+"""
+from ..terminal_verb import TerminalVerb
+
+
+class SpeakSentence(TerminalVerb):
+
+    def __init__(
+        self, text: str, voice: str=None, 
```

Comment:
```suggestion
        self, sentence: str, voice: str=None, 
```
Do we want to call this sentence? I have no preference really but maybe `speakSentence.text = ` is a more logical interface?

Reply:
Shouldn't it match what we have in the docs? https://dev.bandwidth.com/docs/voice/bxml/speakSentence/  I know it's the content so it's not typically seen but if we wanted to change this to sentence I think it should be changed there as well. 

Also, this can be more than one sentence so text might make more sense since there's no length implied.

Reply:
Text is fine with me 👍